src/static/css/toc_adjustments.css

```diff
@@ -7,10 +7,21 @@
     background-color:#f5f5f5;
 }
 
-body{
+body {
     text-align:left !important;
 }
 
+.invalid {
```

Comment:
switch to use language "error / warning / correct"

Reply:
will do